requirements.txt

```diff
@@ -7,9 +7,11 @@ Flask-Bcrypt
 Flask-Login
 flask-mongoengine
 folium
+fsspec
 geopandas
 google-cloud-storage
 google-cloud-firestore
+gcsfs
```

Comment:
Had to add above two packages so `pd.read_csv("gs://<some-gcloud-path>")` worked